### ISSUE TITLE
"No functional TLS backend was found" on Windows Release 2.1.0

### ISSUE THREAD

**Issue by sideshowcecil** [MerginMaps/input#2647]
I just downloaded the latest Windows release 2.1.0 using the win64_build artifact, but when I run the app it cannot connect to any online services and the logs are filled with the following errors:

```
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No TLS backend is available
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No TLS backend is available
qt.network.ssl: No functional TLS backend was found
qt.network.ssl: No functional TLS backend was found
"2023-03-27T05:25:12.446Z Config: Requesting server configuration: https://app.merginmaps.com//config\n"
qt.network.ssl: No functional TLS backend was found
"2023-03-27T05:25:12.446Z ping: Requesting: https://app.merginmaps.com//ping\n"
```

It seems the app is missing the required OpenSSL library in the artifact, as the "tls" folder of the app is empty.

**Comment by tomasMizera** [comment]
Hi @sideshowcecil, Windows build has not been released in 2.1.0 due to this issue. It will be in 2.1.1 later this week

**Comment by sideshowcecil** [comment]
Hi @tomasMizera, thanks for letting me know.
